packages/fast-components-react-base/src/hypertext/README.md

```diff
@@ -1,14 +1,2 @@
-## Best practices
-Use *hypertext* when you need to navigate the user to more information about the text that was selected. *Hypertexts* are best used for tertiary actions outside the main work flow.
-
-### Usage guidance
-*Hypertexts* may appear as inline elements. Use inline *hypertext* within the context of a sentence or a paragraph. Within a paragraph, *hypertext* should inherit the same formatting as the neighboring text as far as font size and font weight. Inline links will appear underlined.
-
-### Style guidance
-Two *hypertexts* can appear consecutively on separate lines of text and wrap. *Hypertext* text must be clear and unique. Multiple *hypertext* to same location should have identical link text.
-
-### Behavioral guidance
-If you need to initiate actions (e.g. download a file), consider using *action trigger* instead. For calls to action (e.g. "buy"), consider using *call to action*, and for the final action in a workflow (e.g. "save"), consider using *button*. Keep *hypertexts* to just a few words, 4-5 max, not entire sentences. Don't include the period if *hypertext* is at the end of a sentence.
-
-## Accessibility
-A *hypertext* is required to have two points of visual distinction when *inline*. The underline and visual state colors are both required for accessibility.
\ No newline at end of file
+## Guidance
+The *Hypertext* component is an [anchor](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) element which accepts children. *Hypertexts* most commonly appear as inline elements within the context of a sentence or a paragraph.
```

Comment:
I feel strongly we should **not** be capitalizing our components.

Reply:
Since documentation is part of this, I will update here. We should document this somewhere so it's explicit.

Reply:
resolved. With exception of those that start a sentence.